contrib/PKGBUILD

```diff
@@ -27,7 +27,14 @@ build() {
     if [ -n "$CI" ]; then
         export CI="--wrap-mode=default"
     fi
-    arch-meson -D b_lto=false $CI ../build -Dsupported_build=true
+    arch-meson -D b_lto=false $CI ../build \
+        -Defi_sbat_distro_id="arch" \
+        -Defi_sbat_distro_generation=1 \
+        -Defi_sbat_distro_summary="Arch Linux" \
+        -Defi_sbat_distro_pkgname="${pkgname}" \
+        -Defi_sbat_distro_version="$(pkgver)" \
+        -Defi_sbat_distro_url="https://archlinux.org/packages/community/x86_64/${pkgname}/" \
+        -Dsupported_build=true
```

Comment:
@superm1 what do you think these values should be for the snap package? Maybe we just don't write a distro section?

Reply:
Well in the snap package it's actually pulling the Ubuntu signed binary at the moment...

Reply:
Ahh, well that makes things simpler then! Thanks.